src/models/pokemon-location.js

```diff
@@ -0,0 +1,53 @@
+@@ -0,0 +1,52 @@
+import { throttle } from 'lodash'
+import axios from 'axios'
+import { observable, asMap } from 'mobx'
+import moment from 'moment'
+
+const serverStatus = observable([ 'unknown', 'default' ])
+const isLoading = observable(false)
+const spawns = asMap({})
+var counter = 0
+
+const STATUSES = [
+  [ 'online', 'success' ],
+  [ 'offline', 'danger' ]
+]
+
+const checkPokemonSpawns = throttle(async (latitude, longitude) => {
+  counter++;
+  isLoading.set(true);
+  console.log('checking spawns')
+  const url = `https://pokevision.com/map/data/${latitude}/${longitude}`
+
+  try {
+    const {data, status} = await axios.get(url);
+    if (status == 200 && data.status === "success") {
+      if (counter > 5) {
+        counter = 0;
+        spawns.clear();
```

Comment:
Don't really like this, there should be a better, optimized way to refresh the mobx map.